homeassistant/components/statistics/sensor.py

```diff
@@ -52,6 +52,7 @@
 DEFAULT_SIZE = 20
 DEFAULT_PRECISION = 2
 ICON = "mdi:calculator"
+UNIT_OF_MEASUREMENT_BINARY = "value"
```

Comment:
The measurement represents the number of state changes within the sample interval, right? Should we use a unit of measurement that better matches that? `"value"` is very generic. Eg: `"changes"` or `"states"`?

Reply:
Local testing shows using `changes` has the same desired effect on the UI as `value` where it's treated like a number by default.